atomic-app/templates/rake-cronjobs.yaml

```diff
@@ -0,0 +1,39 @@
+{{- if .Values.rakeCronjobs.enabled -}}
+{{- $top := . -}}
+{{- range $cronjob := .Values.rakeCronjobs.jobs -}}
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ template "app.fullname" $top }}-{{ $cronjob.name }}
+  labels:
+    {{- include "app.labels" $top | nindent 4 }}
+  namespace: {{ $top.Release.Namespace }}
+spec:
+  schedule: "{{ $cronjob.schedule }}"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          restartPolicy: OnFailure
```

Comment:
I think we should add more from other deployments:

serviceAccountName
securityContext
imagePullPolicy
imagePullSecrets
shared mount (/shared)
resources  (use a new Values.cron maybe?)  This is probably pretty important so k8s know how to schedule these.

whatever else looks useful